cypher-shell/src/integration-test/java/org/neo4j/shell/MainIntegrationTest.java

```diff
@@ -133,9 +139,54 @@ public void wrongPortWithNeo4j() throws Exception
         main.connectMaybeInteractively( shell, connectionConfig, true, true );
     }
 
+    @Test
+    public void shouldReadCypherStatementsFromFile() throws Exception {
```

Comment:
I would like to see tests that you can get prompted for username and/or password from stdin when a file was provided.

Reply:
But that is what is tested here?